controller/instance_manager_controller.go

```diff
@@ -842,12 +843,13 @@ func (imc *InstanceManagerController) syncInstanceManagerPDB(im *longhorn.Instan
 	}
 
 	if clusterAutoscalerEnabled {
-		canDeletePDB, err := imc.canDeleteInstanceManagerPDB(im)
+		canDeletePDB, msg, err := imc.canDeleteInstanceManagerPDB(im)
 		if err != nil {
 			return err
 		}
 
 		if !canDeletePDB {
+			imc.logger.Infof("Node %v is marked unschedulable but removing %v PDB get message: %s. Autoscaler skips it", im.Name, imc.controllerID, msg)
```

Comment:
```suggestion
			imc.logger.Debugf("Autoscaler is enabled but removing %v PDB for node %v is blocked: %v", im.Name, imc.controllerID, msg)
```

Move to `Debugf` because this message will be logged every 30 s when `kubernetes-cluster-autoscaler-enabled == true`, even when nothing interesting is going on. We have had to scale back a lot of similar log messages in the past to keep the support bundle useful.

```
eweber@laptop:~/longhorn> kubetail -n longhorn-system -l app=longhorn-manager
Will tail 6 logs...
longhorn-manager-9j68h longhorn-manager
longhorn-manager-9j68h pre-pull-share-manager-image
longhorn-manager-b9qwf longhorn-manager
longhorn-manager-b9qwf pre-pull-share-manager-image
longhorn-manager-t66qs longhorn-manager
longhorn-manager-t66qs pre-pull-share-manager-image
[longhorn-manager-b9qwf longhorn-manager] time="2024-08-07T13:37:10Z" level=info msg="Node instance-manager-699da83c0e9d22726e667344227e096b is marked unschedulable but removing eweber-v126-worker-9c1451b4-kgxdq PDB get message: some volumes are still attached. Autoscaler skips it" func="controller.(*InstanceManagerController).syncInstanceManagerPDB" file="instance_manager_controller.go:852" controller=longhorn-instance-manager node=eweber-v126-worker-9c1451b4-kgxdq
[longhorn-manager-b9qwf longhorn-manager] time="2024-08-07T13:37:40Z" level=info msg="Node instance-manager-699da83c0e9d22726e667344227e096b is marked unschedulable but removing eweber-v126-worker-9c1451b4-kgxdq PDB get message: some volumes are still attached. Autoscaler skips it" func="controller.(*InstanceManagerController).syncInstanceManagerPDB" file="instance_manager_controller.go:852" controller=longhorn-instance-manager node=eweber-v126-worker-9c1451b4-kgxdq
[longhorn-manager-b9qwf longhorn-manager] time="2024-08-07T13:38:10Z" level=info msg="Node instance-manager-699da83c0e9d22726e667344227e096b is marked unschedulable but removing eweber-v126-worker-9c1451b4-kgxdq PDB get message: some volumes are still attached. Autoscaler skips it" func="controller.(*InstanceManagerController).syncInstanceManagerPDB" file="instance_manager_controller.go:852" controller=longhorn-instance-manager node=eweber-v126-worker-9c1451b4-kgxdq
[longhorn-manager-t66qs longhorn-manager] 10.42.2.0 - - [07/Aug/2024:13:38:14 +0000] "GET /v1/volumes/pvc-99ca718f-a790-46cf-9660-116b761ae0f3 HTTP/1.1" 200 10799 "" "Go-http-client/1.1"
[longhorn-manager-b9qwf longhorn-manager] time="2024-08-07T13:38:40Z" level=info msg="Node instance-manager-699da83c0e9d22726e667344227e096b is marked unschedulable but removing eweber-v126-worker-9c1451b4-kgxdq PDB get message: some volumes are still attached. Autoscaler skips it" func="controller.(*InstanceManagerController).syncInstanceManagerPDB" file="instance_manager_controller.go:852" controller=longhorn-instance-manager node=eweber-v126-worker-9c1451b4-kgxdq
[longhorn-manager-b9qwf longhorn-manager] time="2024-08-07T13:39:10Z" level=info msg="Node instance-manager-699da83c0e9d22726e667344227e096b is marked unschedulable but removing eweber-v126-worker-9c1451b4-kgxdq PDB get message: some volumes are still attached. Autoscaler skips it" func="controller.(*InstanceManagerController).syncInstanceManagerPDB" file="instance_manager_controller.go:852" controller=longhorn-instance-manager node=eweber-v126-worker-9c1451b4-kgxdq
[longhorn-manager-t66qs longhorn-manager] 10.42.2.0 - - [07/Aug/2024:13:39:31 +0000] "GET /v1/volumes/pvc-99ca718f-a790-46cf-9660-116b761ae0f3 HTTP/1.1" 200 10799 "" "Go-http-client/1.1"
...
```

Remove the bit about unscheduable because it does not gate this log.